### PR TITLE
Add links to relevant Learn guides throughout the TFC docs

### PR DESCRIPTION
**Pull request by nfagerlund**
## PR Objective

- [x] Improve navigation by exposing relevant tutorials

image

image

### CODE REVIEW

File: content/source/docs/cloud/sentinel/examples.html.md [hunk 1]
```diff
@@ -7,9 +7,11 @@ page_title: "Example Policies - Policies - Terraform Cloud and Terraform Enterpr
 
 -> **Note:** Terraform policies are a paid feature, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
 
+> For a hands-on tutorial, try the [Enforce Policy with Sentinel](https://learn.hashicorp.com/terraform/sentinel/sentinel-intro?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) guide on HashiCorp Learn.
```
Comment (robin-norwood) [inline content/source/docs/cloud/sentinel/examples.html.md:10]:
```suggestion
> For a hands-on tutorial, try the [Enforce Policy with Sentinel](https://learn.hashicorp.com/terraform/sentinel/sentinel-intro?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) track on HashiCorp Learn.
```

File: content/source/docs/cloud/sentinel/index.html.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ page_title: "Sentinel - Terraform Cloud and Terraform Enterprise"
 
 -> **Note:** Sentinel policies are a paid feature, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
 
+> For a hands-on tutorial, try the [Enforce Policy with Sentinel](https://learn.hashicorp.com/terraform/sentinel/sentinel-intro?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) guide on HashiCorp Learn.
```
Comment (robin-norwood) [inline content/source/docs/cloud/sentinel/index.html.md:10]:
```suggestion
> For a hands-on tutorial, try the [Enforce Policy with Sentinel](https://learn.hashicorp.com/terraform/sentinel/sentinel-intro?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) track on HashiCorp Learn.
```

File: content/source/docs/cloud/sentinel/manage-policies.html.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ page_title: "Managing Sentinel Policies - Sentinel - Terraform Cloud and Terrafo
 
 -> **Note:** Sentinel policies are a paid feature, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
 
+> For a hands-on tutorial, try the [Enforce Policy with Sentinel](https://learn.hashicorp.com/terraform/sentinel/sentinel-intro?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) guide on HashiCorp Learn.
```
Comment (robin-norwood) [inline content/source/docs/cloud/sentinel/manage-policies.html.md:10]:
```suggestion
> For a hands-on tutorial, try the [Enforce Policy with Sentinel](https://learn.hashicorp.com/terraform/sentinel/sentinel-intro?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) track on HashiCorp Learn.
```

File: content/source/docs/cloud/workspaces/creating.html.md [hunk 4]
```diff
@@ -5,6 +5,8 @@ page_title: "Creating Workspaces - Workspaces - Terraform Cloud and Terraform En
 
 # Creating Workspaces
 
+> For a hands-on introduction, try the [Get Started - Terraform Cloud guide](https://learn.hashicorp.com/terraform/cloud-getting-started/signup) on HashiCorp Learn.
```
Comment (robin-norwood) [inline content/source/docs/cloud/workspaces/creating.html.md:8]: The other examples have "guide" outside the link.

```suggestion
> For a hands-on introduction, try the [Get Started - Terraform Cloud](https://learn.hashicorp.com/terraform/cloud-getting-started/signup) guide on HashiCorp Learn.
```

File: content/source/docs/cloud/workspaces/index.html.md [hunk 5]
```diff
@@ -5,6 +5,8 @@ page_title: "Workspaces - Terraform Cloud and Terraform Enterprise"
 
 # Workspaces
 
+> For a hands-on tutorial, try the [Get Started — Terraform Cloud](https://learn.hashicorp.com/terraform/cloud-getting-started/signup?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) guide on HashiCorp Learn.
```
Comment (robin-norwood) [inline content/source/docs/cloud/workspaces/index.html.md:8]:
```suggestion
> For a hands-on tutorial, try the [Get Started — Terraform Cloud](https://learn.hashicorp.com/terraform/cloud-getting-started/signup?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) track on HashiCorp Learn.
```

File: content/source/docs/cloud/workspaces/creating.html.md (hunk 4 above)
Comment (nfagerlund) [reply content/source/docs/cloud/workspaces/creating.html.md:8]: Ah, good catch.

File: content/source/docs/cloud/registry/using.html.md [hunk 6]
```diff
@@ -5,6 +5,8 @@ page_title: "Using Private Modules - Private Module Registry - Terraform Cloud a
 
 # Using Modules from the Terraform Cloud Private Module Registry
 
+> For a hands-on tutorial, try the [Use Terraform Modules](https://learn.hashicorp.com/terraform/modules/using-modules?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) guide on HashiCorp Learn.
```
[the remark this answers is not in the document]
Comment (nfagerlund) [reply content/source/docs/cloud/registry/using.html.md:8]: Hmm, maybe. I'm going to leave it be for now, but we can iterate later.

**Comment by nfagerlund** [comment]
@robin-norwood I'm skeptical that most readers have much interest in what a "track" is (I just think of it as a multi-part guide, tbh), but you're right, so why not. (Besides, we'll need to adjust these later anyway once Collections go in and all the URLs change.)

**Pull request merged**